src/utils.js

```diff
@@ -3,109 +3,101 @@
+    }
+
+    if (!Array.isArray(events)) events = [events];
+
+    for (const element of elements) {
+        for (const event of events) {
+            element[method](event, fn, {capture: false, ...options});
+        }
+    }
+
+    return Array.prototype.slice.call(arguments, 1);
+}
 
 /**
  * Add event(s) to element(s).
  * @param elements DOM-Elements
  * @param events Event names
  * @param fn Callback
  * @param options Optional options
- * @return IArguments passed arguments
+ * @return Array passed arguments
  */
-export function on(elements, events, fn, options = {}) {
-    eventListener(elements, events, fn, options, 'addEventListener');
-    return arguments;
-}
+export const on = eventListener.bind(null, 'addEventListener');
 
 /**
  * Remove event(s) from element(s).
  * @param elements DOM-Elements
  * @param events Event names
  * @param fn Callback
  * @param options Optional options
- * @return IArguments passed arguments
+ * @return Array passed arguments
  */
-export function off(elements, events, fn, options = {}) {
-    eventListener(elements, events, fn, options, 'removeEventListener');
-    return arguments;
-}
-
-function eventListener(elements, events, fn, options = {}, method) {
-
-    // Normalize array
-    if (HTMLCollection.prototype.isPrototypeOf(elements) ||
-        NodeList.prototype.isPrototypeOf(elements)) {
-        elements = Array.from(elements);
-    } else if (!Array.isArray(elements)) {
-        elements = [elements];
-    }
-
-    if (!Array.isArray(events)) {
-        events = [events];
-    }
-
-    for (let element of elements) {
-        for (let event of events) {
-            element[method](event, fn, {capture: false, ...options});
-        }
-    }
-}
+export const off = eventListener.bind(null, 'removeEventListener');
 
 /**
  * Binds all functions, wich starts with an underscord, of a es6 class to the class itself.
  * @param context The context
  */
 export function bindClassUnderscoreFunctions(context) {
     const methods = Object.getOwnPropertyNames(Object.getPrototypeOf(context));
-    for (let fn of methods) {
+    for (const fn of methods) {
         if (fn.charAt(0) === '_' && typeof context[fn] === 'function') {
             context[fn] = context[fn].bind(context);
         }
     }
 }
 
+const unitify = (val, unit = 'px') =>
+    typeof val === 'number' ? val + unit : '' + val;
+
 /**
  * Add css to a DOM-Element or returns the current
  * value of a property.
  *
- * @param el The DOM-Element.
+ * @param el The Element.
  * @param attr The attribute or a object which holds css key-properties.
  * @param val The value for a single attribute.
  * @returns {*}
  */
 export function css(el, attr, val) {
     const style = el && el.style;
+    if (!style) return;
+    
+    if (typeof attr === 'object') {
 
-    if (style) {
-
-        if (typeof attr === 'object') {
+        for (const prop in attr) {
+            const val = attr[prop];
+            style[prop] = unitify(val);
```

Comment:
just realized it could actually just be a one liner. ``style[prop] = unitify(attr[prop])``. oh well no biggie.